server/lists/adapters/__tests__/case-notes.spec.js

```diff
@@ -28,10 +28,10 @@ describe('Case Notes Adapter', () => {
         const mockData = [
             { editedTime: '2019-01-03 14:00:00.571 +000', editedName: 'User A', eventTime: '2019-01-01 12:00:00', type: 'MANUAL', userName: 'User A', body: { caseNote: 'A test case note', userUUID: 1, teamUUID: 1, stage: 1 }, timelineItemUUID: '__timelineItemUUID__' },
             { eventTime: '2019-01-03 12:00:00', type: 'STAGE_ALLOCATED_TO_USER', userName: 'User A', body: { caseNote: 'A test case note', userUUID: 1, teamUUID: 1, stage: 1 }, timelineItemUUID: '__timelineItemUUID__' },
-            { eventTime: '2019-01-05 12:00:01', type: 'STAGE_ALLOCATED_TO_TEAM', userName: 'User A', body: { caseNote: 'A test case note', userUUID: 1, teamUUID: 1, stage: 1 }, timelineItemUUID: '__timelineItemUUID__' },
-            { eventTime: '2019-01-05 12:00:02', type: 'STAGE_CREATED', userName: 'User A', body: { caseNote: 'A test case note', userUUID: 1, teamUUID: 1, stage: 1 }, timelineItemUUID: '__timelineItemUUID__' },
-            { eventTime: '2019-01-05 12:00:03', type: 'STAGE_COMPLETED', userName: 'User A', body: { caseNote: 'A test case note', userUUID: 1, teamUUID: 1, stage: 1 }, timelineItemUUID: '__timelineItemUUID__' },
-            { eventTime: '2019-01-05 12:00:04', type: 'STAGE_RECREATED', userName: 'User A', body: { caseNote: 'A test case note', userUUID: 1, teamUUID: 1, stage: 1 }, timelineItemUUID: '__timelineItemUUID__' },
+            { eventTime: '2019-01-05 12:00:05', type: 'STAGE_ALLOCATED_TO_TEAM', userName: 'User A', body: { caseNote: 'A test case note', userUUID: 1, teamUUID: 1, stage: 1 }, timelineItemUUID: '__timelineItemUUID__' },
```

Comment:
What issue caused these case-note tests to need changing?

Reply:
Some of the records in the array of case-notes have identical eventTimes and that is the field used for ordering. When upgrading to Node14 these records get ordered in a different way by default. So my fix here is to give unique event times to each row, so the order is defined. See https://homeofficedigital.slack.com/archives/C01GS5FD116/p1612799524139000 for the team chatting about this issue.

Reply:
Makes sense and a good solution. Approved.